src/Microsoft.Extensions.Configuration.Abstractions/IConfiguration.cs

```diff
@@ -35,6 +35,10 @@ public interface IConfiguration
         /// <returns>The configuration sub-sections.</returns>
         IEnumerable<IConfigurationSection> GetChildren();
 
+        /// <summary>
+        /// Returns a <see cref="IChangeToken"/> that can be used to obseve when this configuration is reloaded.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
No, we shouldn't. However, `returns` is only shown in Object Browser and on MSDN, not IntelliSense. So it really just depends on what intended targets for documentation are.

I've got [an EF branch](https://github.com/aspnet/EntityFramework/compare/dev...bricelam:style) where I'm enabling some of [StyleCop.Analyzers](https://github.com/DotNetAnalyzers/StyleCopAnalyzers)'s more useful rules to catch things like this.